File: Arrays/Linear_search/sol.py
def search(arr,num):
    for i in range(len(arr)):
        if arr[i] == num:
            return i

    return -1


if __name__ == "__main__":
    arr = [1,2,3,4,5,6]
    num = 3
    print(search(arr,num))

File: Arrays/Linear_search/sol.cpp
#include<stdio.h>

int search(int arr[], int n, int num){
    for(int i=0; i<n; i++){
        if(arr[i] == num){
            return i;
        }
    }

    return -1;
}

int main(){
    int arr[] = {1,2,3,4,5,6,7};
    int n = sizeof(arr)/ sizeof(arr[0]);
    int val = search(arr,n,5);
    printf("%d",val);

}
